### PR TITLE
fix(style): Cleanup sync options styling

### LINKED ISSUE #3394: Choose what to sync options clean-up
choose

1. Remove focus on first checkbox
1. Make checkbox labels less selectable
1. Make checkboxes align with copy (perhaps set the line-height of .fxa-checkbox__label to 20px?)
1. Add down-state depicted below
1. Watch Ryan squeal with glee. So thrilled to be launching this feature.
 
checkbox

### PR DESCRIPTION
**Pull request by vbudhram**
The PR fixes #3394 , however the first input field functionality was not changed because of #3342. 

Update version of fxa-checkbox to have correct styling states.

@vladikoff @shane-tomlinson @philbooth What are your thoughts? r?

### CODE REVIEW

File: app/styles/modules/_custom-rows.scss [hunk 1]
```diff
@@ -86,6 +86,16 @@ ul.links {
 }
 
 .choose-what-to-sync-row {
+  &:active{
+    .fxa-checkbox__box-outline{
+      border: 2px solid $checkbox-border-color;
+
+      .fxa-checkbox__tick-outline{
+        background: url("/images/tick.svg");
```
Comment (vladikoff) [inline app/styles/modules/_custom-rows.scss]: @vbudhram can we include these changes in fxa-checkbox instead?  feel free to remove the encoded svg in it and replace it with a new one

File: app/images/tick.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1"
```
Comment (vladikoff) [inline app/images/tick.svg]: this file will need to be optimized with svgo

File: app/styles/modules/_custom-rows.scss (hunk 1 above)
Comment (vladikoff) [inline app/styles/modules/_custom-rows.scss]: https://github.com/mozilla/fxa-checkbox

Comment (vbudhram) [inline app/styles/modules/_custom-rows.scss]: Sounds good, created [Issue](https://github.com/mozilla/fxa-checkbox/issues/4) to track this.

Comment (vladikoff) [inline app/styles/modules/_custom-rows.scss]: > Sounds good, created Issue to track this.

:+1: we can close this pr and move it into there. then the content server PR will be just updating bower.json

Comment (vbudhram) [inline app/styles/modules/_custom-rows.scss]: I'm good with that, I will see if I can also fix the label alignment in fxa-checkbox as well.

**Comment by vladikoff** [comment]
@vbudhram Latest SHA available here: https://github.com/mozilla/fxa-checkbox/commits/master to use with this PR

**Comment by vbudhram** [comment]
Thanks @vladikoff, updated version, review when ready :)

**Comment by shane-tomlinson** [comment]
@vbudhram - can you rebase this branch?

**Comment by vbudhram** [comment]
@shane-tomlinson done!

**Comment by vladikoff** [comment]
:+1:

**Pull request merged**